index.html

```diff
@@ -2501,8 +2503,8 @@ <h4 id="multibase">The `multibase` Datatype</h3>
           <dt>The lexical-to-value mapping</dt>
           <dd>
 Any element of the lexical space is mapped to the value space by base-decoding
-the value based on the base-decoding alphabet associated with the
-first <a href="#multibase">Multibase</a> character in the lexical string.
+the value based on the base-decoding alphabet associated with the first
+[=Multibase header=] in the lexical string.
```

Comment:
Can there be more than one Multibase header in such a lexical string? Or is `first` unnecessary here?

Reply:
It might be more accurate to using the first character of the lexical string as the multibase header. I think this text was intended to make it clear that while the lexical string is composed of many characters (of course), it is the first one (the first character) that expresses the multibase header.